packages/js/data/src/onboarding/actions.ts

```diff
@@ -527,7 +527,8 @@ export function* actionTask( id: string ) {
 }
 
 export function* installAndActivatePluginsAsync(
-	plugins: Partial< PluginNames >[]
+	plugins: Partial< PluginNames >[],
+	source?: string
```

Comment:
I think it would be beneficial to include some documentation/comments somewhere in the code to explain the usage of the "source".

Reply:
@chihsuan Good catch! Fixed it in [Document `source](https://github.com/woocommerce/woocommerce/pull/56373/commits/5d5643887262e8272b6928f5992d7af3cbf4f328)